subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -38,6 +38,11 @@
 
 #include "ull_conn_internal.h"
 #include "ull_conn_types.h"
+
+#if defined(CONFIG_BT_CTLR_USER_EXT)
+#include "ull_vendor.h"
+#endif /* CONFIG_BT_CTLR_USER_EXT */
```

Comment:
Ditto

Reply:
This is taken from the legacy Connection Parameter Request LLCP procedure https://github.com/zephyrproject-rtos/zephyr/blob/f6a3da9c21806364be6c9fd98079204c5f2b28f6/subsys/bluetooth/controller/ll_sw/ull_conn.c#L45-L47 to support vendor extension control over the value returned by `CONN_INTERVAL_MIN()` in 

Legacy:
https://github.com/zephyrproject-rtos/zephyr/blob/f6a3da9c21806364be6c9fd98079204c5f2b28f6/subsys/bluetooth/controller/ll_sw/ull_conn.c#L7102 and 

New:
https://github.com/zephyrproject-rtos/zephyr/blob/20cb405e17895955d97d0eca0b23fe26180f321f/subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c#L177